Stratis.Bitcoin/Wallet/WalletExtensions.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using NBitcoin;
+
+namespace Stratis.Bitcoin.Wallet
+{
+    public static class WalletExtensions
+    {
+	    public static List<TransactionData> UnspentTransactions(this HdAddress address)
```

Comment:
This doesn't take into accounts the change addresses.
Additionally, there is a method in Wallet.HdAccount called "public IEnumerable<TransactionData> GetSpendableTransactions()" which you may want to use instead.